omero/developers/testing.rst

```diff
@@ -280,32 +280,32 @@ tests from :sourcedir:`components/tools/OmeroPy` using
 Since Python is interpreted, tests can be written and then run without having
 to rebuild or restart the server. A few basic options are shown below.
 
-.. program:: setup.py test
+First install dependencies::
+
+    $ pip install pytest mox3 pyyaml
 
-.. option:: -t <test_path>, --test-path <test_path>
+    # for Omeroweb tests
+    $ pip install pytest-django
 
-    This option specifies the test suite to run. For instance to run a single
-    test file::
+Run tests directly with pytest, setting the :envvar:`ICE_CONFIG` as described above:
 
-        cd components/tools/OmeroPy
-        ./setup.py test -t test/integration/test_admin.py
+    export ICE_CONFIG=/path/to/openmicroscopy/etc/ice.config
```

Comment:
Something's gone wrong with the formatting, this doesn't appear as a code block:


<img width="595" alt="Screen Shot 2020-01-23 at 11 24 25" src="https://user-images.githubusercontent.com/1644105/72980706-fcaf6100-3dd2-11ea-8557-74b4c984a6fc.png">

Reply:
Also there's another `ice.config` under `OmeroPy`? I thought the tests automatically use this without needing to set `ICE_CONFIG`?
```
etc/ice.config
components/tools/OmeroPy/ice.config
```

Reply:
This may have been lost from the migration away from `omero_setup.py`. If so, we can add it back in, but it should be in the most pytest-y way.

Reply:
I have tried to run a test without setting the Ice_CONFIG
the ``ice.config`` under ``OmeroPy`` did not get picked up.

Reply:
Formatting looks OK to me at https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-docs/lastSuccessfulBuild/artifact/omero/_build/html/developers/testing.html#running-tests-directly and it seems we do need to set ICE_CONFIG so I'll leave it as it is.